src/Connector.SqlServer/Connector/SqlServerConnector.cs

```diff
@@ -253,16 +243,54 @@ public async Task VerifyExistingContainer(ExecutionContext executionContext, Str
             }
 
             await Task.WhenAll(tasks);
+
+            async Task createTable(SanitizedSqlName tableName, IEnumerable<ConnectionDataType> columns, IEnumerable<string> keys,
```

Comment:
agree about separate internal methods or even just a straightforward method body instead of local functions. Unfortunately haven't had time to properly refactor this.
At least not in scope of this task.